tensorflow-framework/src/main/java/org/tensorflow/framework/metrics/impl/MetricsHelper.java

```diff
@@ -15,20 +15,23 @@
 package org.tensorflow.framework.metrics.impl;
 
 import org.tensorflow.Operand;
+import org.tensorflow.framework.losses.impl.LossTuple;
+import org.tensorflow.framework.losses.impl.LossesHelper;
 import org.tensorflow.framework.metrics.exceptions.NotBroadcastableException;
+import org.tensorflow.framework.utils.SparseTensor;
 import org.tensorflow.ndarray.Shape;
 import org.tensorflow.op.Op;
 import org.tensorflow.op.Ops;
+import org.tensorflow.op.core.Stack;
+import org.tensorflow.op.core.*;
 import org.tensorflow.op.math.Mean;
-import org.tensorflow.types.TBool;
-import org.tensorflow.types.TFloat64;
-import org.tensorflow.types.TInt32;
+import org.tensorflow.op.nn.TopK;
+import org.tensorflow.types.*;
 import org.tensorflow.types.family.TIntegral;
 import org.tensorflow.types.family.TNumber;
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
```

Comment:
No star imports.

Reply:
OK, I had to change the default setting in IntelliJ.